.github/workflows/label.yml

```diff
@@ -0,0 +1,28 @@
+
+# This workflow will triage pull requests and apply a label based on the
+# paths that are modified in the pull request.
+#
+# To use this workflow, you will need to set up a .github/labeler.yml
+# file with configuration.  For more information, see:
+# https://github.com/actions/labeler/blob/master/README.md
+
+name: PR Labeler
+on:
+  pull_request:
+    types: [opened, synchronize]
+jobs:
+  pr-labeler:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: codelytv/pr-size-labeler@v1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          xs_max_size: '100'
+          s_max_size: '500'
```

Comment:
I feel like adding a new component would be a medium. For example, when you added the new Hero component
https://github.com/SU-SWS/decanter/pull/259
It's around 300 lines changed which would make it a "small". I'm not too concerned as long as we all understand it in the same way like the JIRA ticket sizing 😄  Just want to bring that up as an example. Otherwise good to go.

Reply:
For this project I made the default settings a little smaller. If we do change a 1000 lines of code or more I would suspect it is an XL review. The labels don't mean much other than a t-shirt size of how much code has changed.